db/schema.rb

```diff
@@ -105,22 +105,22 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "usuarios", force: :cascade do |t|
+  create_table "users", force: :cascade do |t|
     t.string "email", default: "", null: false
     t.string "encrypted_password", default: "", null: false
     t.string "reset_password_token"
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["email"], name: "index_usuarios_on_email", unique: true
-    t.index ["reset_password_token"], name: "index_usuarios_on_reset_password_token", unique: true
+    t.index ["email"], name: "index_users_on_email", unique: true
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
```

Comment:
Did you edit manually the `schema.rb` file?
This file should be modified automatically by running the migrations. That said, I didn't see a migration where you changed the indexes. Were these indexes affected by your current migrations only?

Reply:
No, I modified them because I wasn't sure about rolling back

Reply:
The modifications that I made manually, can I revert them manually? Or will it cause a problem like in this case?

Reply:
I will do it again

Reply:
Just run your migrations again and keep the schema as it will be.
Then, if there is still any index with `usuario`, let us know, so we can create a migration to fix it.

Reply:
I already modified it, that's it and it works :D